corehq/apps/app_manager/xform.py

```diff
@@ -948,6 +948,16 @@ def get_questions(self, langs, include_triggers=False,
                     "type": "DataBindOnly",
                     "calculate": bind.attrib.get('calculate') if hasattr(bind, 'attrib') else None,
                 }
+
+                # Include meta information about the stock entry
+                if data_node.tag_name == 'entry':
+                    parent = next(data_node.xml.iterancestors())
+                    if parent and WrappedNode(parent).tag_name in STOCK_QUESTION_TAG_NAMES:
+                        question.update({
+                            "stock_entry_attributes": data_node.xml.attrib,
+                            "stock_type_attributes": parent.attrib,
+                        })
```

Comment:
this is super paranoid but do we want to use the xmlns (`"http://commcarehq.org/ledger/v1"`) to confirm this is actually a ledgerxml block? in theory someone could make a group called `balance` and stick a question called `entry` inside.